### ISSUE TITLE
Traceback is printed in jupyter notebook before the plot is displayed properly

### ISSUE THREAD

**Issue by gtauzin** [holoviz/holoviews#5008]
First of all, thank you for creating such an amazing python visualization library!

#### Is your feature request related to a problem? Please describe.
When I create a plot, a spurious traceback is printed which is unintelligible to me. Then the plot is properly displayed after that

#### Describe the solution you'd like
If there is an actual error, it seems to me it should be raised. Otherwise, it could be printed as a warning so that I can filter it. Being printed like that just fills up my jupyter cell.

#### Describe alternatives you've considered
Modifying the code to comment out the printing of the traceback such as in https://github.com/gtauzin/holoviews/commit/bba8b3363c2e752343d61ccefbbac19a3527599b

Thank you!

**Comment by hoxbro** [comment]
Do you have a MRE?

**Comment by gtauzin** [comment]
Indeed, would have been helpful to provide that. Sry about that.

Unfortunately, it's been quite a few years and I do not have access to the code in question anymore.

**Comment by hoxbro** [comment]
Yes, I didn't have high hopes that you had an MRE after so many years. 

I will close this issue; you are welcome to open a new issue if you encounter the problem again.
